doc/source/dev/contributor/runtests.rst

```diff
@@ -29,33 +29,54 @@ To run tests on a particular submodule, such as ``optimize``, use the
 
    python runtests.py -v -s optimize
 
-To run a particular test module, such as
-|test-linprog|_, use the ``--test`` option:
+To run a particular test module, use the ``--test`` option:
 
 ::
 
-   python runtests.py -v -t scipy/optimize/tests/test_linprog.py
+   python runtests.py -v -t scipy.<module>.tests.<test_file>
 
-To run a test class, such as ``TestLinprogIPDense`` from
-``test_linprog.py``:
+Exemple for |test-linprog|_ file tests, run:
```

Comment:
typo--used French instead of English spelling of "Example"

Reply:
Thank you for your review. Fix in db34472146a1e7a35a9c3a02bd3dc259acb766c0